README.md

```diff
@@ -1,38 +1,56 @@
 # Neural Network Compression Framework (NNCF)
 
-This repository contains a PyTorch\*-based framework and samples for neural networks compression.
+NNCF provides a suite of advanced compression algorithms, applied during a model fine-tuning and samples for models 
```

Comment:
I would add something about optimization for inference. Sometimes people get confused from the beginning. Like, `NNCF provides a suite of advanced algorithms for Neural Networks inference optimization...`.

Reply:
I have reformulated it, please take a look.